sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -247,6 +247,13 @@
                 </goals>
               </execution>
             </executions>
+            <dependencies>
+              <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-core</artifactId>
```

Comment:
I think instead we would use confluent-log4j, 2.17.1-cp2.2 or cp8 here.  cp8 for branches >= 7.0.x
```xml
<confluent-log4j.version>1.2.17-cp8</confluent-log4j.version>
```
and
```xml
        <dependency>
            <groupId>io.confluent</groupId>
            <artifactId>confluent-log4j</artifactId>
            <version>${confluent-log4j.version}</version>
            <scope>test</scope>
        </dependency>
```

Reply:
confluent-log4j is our internal log4j port. There are a bunch of CVE's out to replace all log4j references with our build.

Reply:
nevermind, I was thinking this was the conector.